pdr_backend/pdr_dashboard/assets/styles.css

```diff
@@ -121,6 +121,77 @@ body {
   -webkit-line-clamp: 2;
 }
 
+#feeds_page_table_area:has(> *[data-dash-is-loading="true"]) > #feeds_page_table table td div,
+#feeds_page_table_area:has(> *[data-dash-is-loading="true"]) > #feeds_page_table table td input,
+#predictoors_page_table_area:has(> *[data-dash-is-loading="true"]) > #predictoors_page_table table td div,
+#predictoors_page_table_area:has(> *[data-dash-is-loading="true"]) > #predictoors_page_table table td input 
+{
+    visibility: hidden;
+}
+
+#feeds_page_table_area:has(> *[data-dash-is-loading="true"]) > #feeds_page_table table td,
+#predictoors_page_table_area:has(> *[data-dash-is-loading="true"]) > #predictoors_page_table table td {
+    position: relative;
+    background-color: #f0f0f0;
+    color: transparent; /* Hides the text */
+    overflow: hidden;
+    border-color: transparent !important;
+    border-radius: 5px;
+    cursor: progress;
+}
+
+#feeds_page_table_area:has(> *[data-dash-is-loading="true"]) > #feeds_page_table table td::after,
+#predictoors_page_table_area:has(> *[data-dash-is-loading="true"]) > #predictoors_page_table table td::after {
+    content: "";
+    display: block;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background: linear-gradient(90deg, rgba(255, 255, 255, 0), rgba(255, 255, 255, 0.5), rgba(255, 255, 255, 0));
+    animation: shimmer 2s infinite;
+}
+
+.metric_unit {
+    display: flex;
+    flex-direction: column;
+    font-size: 20px;
+}
+
+#feeds_page_metrics_row_wrapper:has(> *[data-dash-is-loading="true"]) .metric_value {
```

Comment:
I think for the metrics it's better if we wrap the component inside a` dcc.Loading` component, just like the metrics from the home page.
Also on the home page, for the graphs, instead of a displaying a loading component when data is loading, display some empty graphs and a loading message(maybe also text and loading component) just like in the table modals

Reply:
The dcc.Loading does not let a skeleton loading. It is suggested on the dash docs

Reply:
Ok, the target is to use the same loading pattern everywhere, so If you want to use skeleton loading the replace the dcc.Loading. Initially I was thinking on using skeleton just for the table, because we can't use dcc.Loading for the cells